src/functions/helperFiles/meetupHelpers.js

```diff
@@ -0,0 +1,104 @@
+const striptags = require('striptags')
+
+const generateContentfulEvent = ({
+    urlname,
+    nextEvent,
+    venue,
+    link,
+    date,
+    time,
+    duration,
+    eventName,
+    description
+}) => ({
+    fields: {
+        thisMeetupCode: {
+            'en-US': `${urlname}-${nextEvent}`
+        },
+        meetupUrlName: {
+            'en-US': urlname
+        },
+        linkToEvent: {
+            'en-US': link
+        },
+        date: {
+            'en-US': date
+        },
+        startTime: {
+            'en-US': new Date(time)
+        },
+        endTime: {
+            'en-US': new Date(time + duration)
+        },
+        address: {
+            'en-US':
+                venue !== 'Venue To Be Confirmed'
+                    ? `${venue.name}&&${venue.address1}&&${
+                    venue.adress2 ? venue.adress2 : ''
+                    }&&${venue.address3 ? venue.address3 : ''}&&${venue.city}`
+                    : 'Venue To Be Confirmed'
+        },
+        eventTitle: {
+            'en-US': eventName
+        },
+        blurb: {
+            'en-US': description
+        },
+        homepageFeatured: {
+            'en-US': false
+        }
+    }
+})
+const processMeetupData = arrayOfMeetups => {
+    let outputArray = []
+    arrayOfMeetups.forEach(meetup => {
+        const thisMeetup = {
+            meetupId: meetup.id,
+            name: meetup.name,
+            url: meetup.link,
+            urlname: meetup.urlname,
+            nextEvent: meetup.next_event ? meetup.next_event.id : 0
+        }
+
+        outputArray.push(thisMeetup)
```

Comment:
No need to define `const thisMeetup` if used once.
```
outputArray.push({
  meetupId: meetup.id,
  name: meetup.name,
  url: meetup.link,
  urlname: meetup.urlname,
  nextEvent: meetup.next_event ? meetup.next_event.id : 0
})
```

Reply:
Actually better...for each with a push...we can simply `.map`
```
const processMeetupData = arrayOfMeetups => arrayOfMeetups.map(meetup => ({
  meetupId: meetup.id,
  name: meetup.name,
  url: meetup.link,
  urlname: meetup.urlname,
  nextEvent: meetup.next_event ? meetup.next_event.id : 0
}))
```

Reply:
.map is magic!